src/Administration/Resources/app/administration/src/module/sw-plugin/view/sw-plugin-list/index.js

```diff
@@ -355,10 +360,12 @@ Component.register('sw-plugin-list', {
             });
         },
 
-        getList() {
+        getList(searchCriteria) {
             this.isLoading = true;
 
-            return State.dispatch('swPlugin/updatePluginList', this.searchData)
+            this.listingCriteria = searchCriteria;
+
+            State.dispatch('swPlugin/updatePluginList', this.searchData)
```

Comment:
This would break the `getList` method. It returned a Promise before and now it is a void function. This can break plugins :)